### ISSUE TITLE
[BUG] - go get -u error on fetching the project

### ISSUE THREAD

**Issue by sujit-baniya** [fasthttp/websocket#25]
When trying to fetch project, I get following error

```go
go get -u github.com/fasthttp/websocket                        
github.com/fasthttp/websocket imports
        github.com/savsgio/gotils: cannot find module providing package github.com/savsgio/gotils

```

**Comment by alexg-axis** [comment]
The following worked for me:

```
# 1. Clean cache
go clean -modcache
# 2. Check that you don't have conflicting versions (github.com/gofiber/fiber/v2 and github.com/gofiber/fiber) in your go mod
# 3. Tidy
go mod tidy
```

**Comment by sujit-baniya** [comment]
@alexg-axis I tried above and didn't help when tried to do `go get -u github.com/fasthttp/websocket`

**Comment by alexg-axis** [comment]
I got the issue again and this time I tried another solution.

1. Remove all indirect dependencies
2. Remove `go.sum` (be aware of the potential issues this could lead to)
3. Manually add your dependencies, my example pasted below
4. Run `go mod tidy` - don't run `go get -u`

```go
require (
	// ...
	github.com/fasthttp/websocket v1.4.3-rc.9
	github.com/gofiber/fiber/v2 v2.19.0
	github.com/valyala/fasthttp v1.30.0
)
```

See: https://github.com/gofiber/websocket/issues/66.

**Comment by sujit-baniya** [comment]
@alexg-axis The problem is with `go get -u` on the package. Because manually upgrade each dependencies are is not a solution as there will be more dependencies.

**Comment by alexg-axis** [comment]
I understand. It's not a solution, rather a workaround.

**Comment by sujit-baniya** [comment]
@savsgio Any idea for fixing this issue?

**Comment by savsgio** [comment]
Hi all,

The version `v1.4.3` is wrong version, because when i created the first beta version of `v1.4.3` i made the mistake when i wrote the tag version. I will try to contact with go.dev staff to remove the `v1.4.3` version.

So, now the last correct version is `v1.4.3-rc.9`. 

Thanks and sorry!

**Comment by sujit-baniya** [comment]
@savsgio Still the issue is not fixed.

![image](https://user-images.githubusercontent.com/4959215/137099334-5b486d15-23d1-4d27-8d61-19dee8faae43.png)

I think maybe just for the sake to fix the issue, Create a different release tag v1.4.4-rc.1 on new commit?

**Comment by G2G2G2G** [comment]
still
```
go get -u all
go get all: module github.com/savsgio/gotils@upgrade found (v0.0.0-20210921075833-21a6215cb0e4), but does not contain package github.com/savsgio/gotils
```

**Comment by efectn** [comment]
Getting same error.

**Comment by Brian-NgH777** [comment]
Getting same error.

**Comment by G2G2G2G** [comment]
@savsgio give me write permission so I can change the version number

**Comment by savsgio** [comment]
Hi everyone,

I created an issue (https://github.com/golang/go/issues/50129) to remove the invalid `v1.4.3` version.

Thanks for your patience!

**Comment by sujit-baniya** [comment]
@savsgio Maybe just create another version 1.4.4 to fix the error

**Comment by savsgio** [comment]
Hi,

I'be just created the `v1.4.4` version to fix the error, without retract `v1.4.3` due to the retract directive is not compatible with Go 1.15.

Thanks and sorry! 😉
